shopfloor/actions/schema.py

```diff
@@ -109,6 +109,8 @@ def location(self):
             "id": {"required": True, "type": "integer"},
             "name": {"type": "string", "nullable": False, "required": True},
             "barcode": {"type": "string", "nullable": True, "required": False},
+            "operations_done": {"type": "float", "required": False},
+            "operations_to_do": {"type": "float", "required": False},
```

Comment:
I'd wrap this into its own key with `done` and `todo` subkeys.
```suggestion
            "operation_info": {"type": "dict", "required": False, ...},
```